File: b964.1.成績指標/main.cpp
#include<iostream>
#include<vector>
#include<algorithm>
using namespace std;
int main(void)
{
	int n=0,a=0;
	vector<int>m;
	while (cin >> n)
	{
		for (int i = 0; i < n; i++)
		{
			cin >> a;
			m.push_back(a);
		}
		sort(m.begin(), m.end());
		for (int j = 0; j < n; j++)
		{
			cout << m[j] << " ";
		}
		cout << endl;
		int cnt = 0;
		for (int k = 0; k < n; k++)
		{
			if (m[k] >= 60)
			{
				cnt++;
				if (k == 0)
				{
					cout << "best case" << endl;
					cout << m[0] << endl;
					break;
				}
				cout << m[k - 1]<<endl;
				cout << m[k] << endl;
				break;
			}
		}
		if (cnt == 0)
		{
			cout << m[n - 1] << endl;
			cout << "worst case" << endl;
		}
	}
	return(0);
}

File: b964.1.成績指標/main.py
n=input()
num=list(map(int,input().split()))
num.sort()
for i in num:
    print(i,end=" ")
print()
if num[0]>=60:
    print("best case")
    print(num[0])
else:
    if num[len(num)-1]<60:
        print(num[len(num)-1])
        print("worst case")
    else:
        for i in range(len(num)):
            if num[i]>=60:
                print(num[i-1])
                print(num[i])
                break